content/influxdb/cloud-dedicated/process-data/visualize/tableau.md

```diff
@@ -77,7 +77,14 @@ To query {{< cloud-name >}} from Tableau, use the **Flight SQL protocol** and th
       - **Protocol**: `jdbc:arrow-flight-sql`
       - **Port**: `443`
       - **Query parameters**:
-        - **disableCertificateVerification**: `true`
+        - **useSystemTrustStore**: `false`
+
+        {{% note %}}
+        #### Setting `useSystemTrustStore=false` is:
+          1. Only necessary on a mac
+          2. Doesn't actually affect the security
+        {{% /note %}}
```

Comment:
I'm not sure if a note can be put in between bullet points in the docs. Feel free to adjust it if there is a better place to put the note